draft-js-dnd-plugin/src/modifiers/addBlock.js

```diff
@@ -1,7 +1,7 @@
 import { List, Repeat } from 'immutable';
 import { Modifier, CharacterMetadata, BlockMapBuilder, EditorState, ContentBlock, Entity, genKey } from 'draft-js';
 
-export default function (editorState, selection, type, data, text = ' ') {
+export default function (editorState, selection, type, data, entityType = undefined, text = ' ') {
```

Comment:
What's the benefit to set this to undefined by default? Isn't it undefined except it is defined?